plugins/BEdita/Core/src/Model/Table/ProfilesTable.php

```diff
@@ -38,6 +38,8 @@ public function initialize(array $config)
         $this->setPrimaryKey('id');
         $this->setDisplayField('name');
 
+        $this->addBehavior('BEdita/Core.Relations');
```

Comment:
Ok, I checked and on `/users` and `/locations` relations are not shown - so, yes, it has to be added there too - but this raises the other question: what are mandatory behaviors in this table classes? what are not? Ok we can answer later to that :smiley_cat: